mypy-strict.ini

```diff
@@ -6,7 +6,7 @@
 # files.
 
 [mypy]
-python_version = 3.8
+python_version = 3.9
```

Comment:
NIt: Should be it's own PR for easy reversion

Reply:
Because refactor is from ruff